nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -65,6 +65,8 @@ Use `services.pipewire.extraConfig` or `services.pipewire.configPackages` for Pi
 
 - Plasma 6 is now available and can be installed with `services.xserver.desktopManager.plasma6.enable = true;`. Plasma 5 will likely be deprecated in the next release (24.11). Note that Plasma 6 runs as Wayland by default, and the X11 session needs to be explicitly selected if necessary.
 
+- The dbus default implementation is now dbus broker instead of the classic dbus deamon. This should result in better performance and reliability without any incompatibilities.
```

Comment:
Not without ANY imcompatibility.

https://github.com/bus1/dbus-broker/wiki/Deviations

(I suggest we attach this link for people to review)

Reply:
Also we should mention this option so that users can modify it according to their needs.

Reply:
Also we should mention this option so that users can modify it according to their needs.